src/pages/home-page/HomePage.js

```diff
@@ -1,7 +1,14 @@
 import "./home-page.scss";
+import "./../../style/common/common.scss";
+import logo from "../../assets/logo.png";
+import LoginForm from "../../components/users/LoginForm/LoginForm";
 
 const HomePage = () => (
   <div className="home-page">
+    <div className="login">
+      <img src={logo} alt="Logo" className="logo" />
```

Comment:
You can give the logo an empty alt property because it's not relevant to the page. The logo here is just a cosmetic add-on.